silk/static/silk/js/pages/sql.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function () {
+  document.querySelectorAll(".data-row").forEach((rowElement) => {
+    let sqlDetailUrl = rowElement.dataset.sqlDetailUrl;
+    rowElement.addEventListener("mousedown", (e) => {
```

Comment:
think this would be better as a "click" event rather than "mousedown", for example in case someone was trying to highlight text.